spec/models/alert_feed_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe AlertFeed, type: :model do
+  before :each do
+    @region = Region.create!(
```

Comment:
Because I learned rspec a really long time ago and have been using minitest recently :) I can change this if there's a reason to.